pages/index.tsx

```diff
@@ -1,72 +1,89 @@
-import type { NextPage } from 'next'
-import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import type { NextPage } from "next";
+import Head from "next/head";
+import React, { useState } from "react";
+import styles from "../styles/Home.module.css";
 
 const Home: NextPage = () => {
+  type inputType = string;
+  type todoListType = Array<String>;
+
+  const [input, setInput] = useState<inputType>("");
+  const [todoList, setTodoList] = useState<todoListType>([]);
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    setInput(e.target.value);
+  };
+
+  const handleAdd = (e: React.FormEvent): void => {
+    e.preventDefault();
+    if (input) {
+      setTodoList([...todoList, input]);
+      setInput("");
+    }
+  };
+
+  const handleDelete = (index: number): void => {
+    const newTodoList = todoList.filter(
+      (todo) => todoList.indexOf(todo) !== index
```

Comment:
```suggestion
      (todo, _, array) => array.indexOf(todo) !== index
```
mấy hàm này có 3 tham số `(item, index, array)`. Thì lấy tham số thứ 3 là array để xử lý nhé
Còn đặt tên `_` cho những thứ không dùng (ở đây là tham số thứ 2 là index), thì sẽ tránh việc nó thông báo warning là "defined but not used"